### DISCUSSION TITLE
How to build pdo_oci ext from a php-src fork?

### DISCUSSION THREAD

**Discussion by mvorisek** [mlocati/docker-php-extension-installer#504] [Q&A] [answered]
This is a question.

I normally build/install php extensions using this DockerFile: https://github.com/mvorisek/image-php/blob/master/data/8.0-debian/Dockerfile

Is there any simple way how can I use https://github.com/php/php-src/pull/8018/files to build the pdo_oci ext to test the php-src fix?

**Comment by mlocati** [comment, marked as answer]
You could try the following:

1. extract the PHP source code that comes with the PHP docker images with `docker-php-source extract`
2. patch the PHP source code you will find at `/usr/src/php` (you could try to download the patch available at https://github.com/php/php-src/pull/8018.patch and to apply it with the `patch` command)
3. run the `install-php-extensions` command as usual

**Comment by mlocati** [comment]
PS: for general discussions what about using https://github.com/mlocati/docker-php-extension-installer/discussions ?

**Comment by mvorisek** [comment]
thanks for a quick reply, will try later today

> general discussions

we can transfer it there 👍, it can be helpful for others

**Comment by mlocati** [comment]
Done :wink:
